index2.html

```diff
@@ -0,0 +1,1253 @@
+        <li>
+          Make a request to the underlying platform to send <var>output</var>
+          to the <a>NFC peer</a>s in proximity range. If the request fails, then
+          reject <var>promise</var> with a new <code><a>DOMException</a></code>
+          object whose <var>name</var> property is set to
+          <code>"NotSupportedError"</code>, and terminate these steps.
+        </li>
+        <li>
+          If the request is successful, then resolve <var>promise</var>.
+        </li>
+      </ol>
+    </p>
+  </section> <!-- setPushMessage() method -->
+
+  <!-- The clearPushMessage() method -->
+  <section><h3>The <strong>clearPushMessage</strong>() method</h3>
+  <p>
+    When the <code><dfn id="widl-NfcAdapter-clearPushMessage-void">
+    clearPushMessage</dfn>()</code> method is invoked, the
+    <a>user agent</a> MUST run the following steps:
+    <ol id="steps-clearPushMessage">
+      <li>Let <var>promise</var> be a new <a><code>Promise</code></a> object.
+      </li>
+      <li>
+        Return <var>promise</var> and continue the following steps
+        asynchronously.
+      </li>
+      <li>
+        If there are no sufficient permissions to use this method,
+        then reject <var>promise</var> with a new
+        <code><a>DOMException</a></code> object whose <var>name</var> property
+        is set to <code>"<a>SecurityError</a>"</code>,
+        and terminate these steps.
+      </li>
+      <li>
+        Make a request to the underlying platform to remove and cancel pushing
+        any previously set messages to <a>NFC peer</a> devices in range or
+        coming into range.
+      </li>
+      <li>
+        If the request fails, then reject <var>promise</var> with a new
+        <code><a>DOMException</a></code> object whose <var>name</var> property
+        is set to <code>"NotSupportedError"</code>, and terminate these steps.
+      </li>
+      <li>
+        If the request is successful, then resolve <var>promise</var>.
+      </li>
+    </ol>
+  </p>
+  </section> <!-- clearPushMessage() -->
+</section>
+
+<!-- - - - - - - - - - - - - - -  Changes - - - - - - - - - - - - - - - - - -->
+<section class="appendix" id="Changes"><h2>Changes</h2>
+  <p>
+    The following is a list of substantial changes to the document. For a
+    complete list of changes, see the <a href=
+    "https://github.com/w3c/nfc/commits/gh-pages">change log on
+    Github</a>. You can also view the <a href=
+    "https://github.com/w3c/nfc/issues?page=1&amp;state=closed">
+    recently closed bugs</a>.
+  </p>
+  <ul>
+    <li>Redesigned the API to follow contemporary web design patterns</li>
+    <li>Change of editors</li>
+  </ul>
+</section>
+
+
+<!-- - - - - - - - - - - - - - - Acknowledgements - - - - - - - - - - - - - -->
+<section> <h2>Acknowledgements</h2>
+  <p>
+    The editors would like to express their gratitude to the former editors
+    Luc Yriarte and Samuel Ortiz, and also to Don Coleman and Salvatore Iovene
```

Comment:
I didn't know about Marcos. if you did, you should have added. Did you tell me to add him? Don't remember. What older version did you mean?